src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/childStatusDescription.jsx

```diff
@@ -0,0 +1,49 @@
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import React from 'react';
+
+export const childStatusDescription = (
+  <div>
+    <p className="vads-u-font-size--h3 vads-u-color--gray-dark vads-u-font-weight--bold vads-u-margin-bottom--1p5">
+      Additional evidence needed
+    </p>
+    <p>
+      If you select a option other than{' '}
+      <span className="vads-u-color--gray-dark vads-u-font-weight--bold">
+        Biological
+      </span>{' '}
+      or if you don't live in the U.S. or a territory of the U.S., we require
+      additional evidence to establish a dependent.
+    </p>
+    <AdditionalInfo triggerText="What are some examples of additional evidence?">
+      <p>What are some examples of additional evidence?</p>
+      <p>
+        You’ll need to provide a copy of your stepchild’s birth certificate,
+        showing the names of both parents if you don’t live in the U.S. or a
+        territory of the U.S.
+      </p>
+      <p>
+        A claim for additional benefits for an adopted child must include a copy
```

Comment:
`adopted child` should be bold.

Edit: [mockup reference](https://xd.adobe.com/view/0430be78-21b6-46e2-68b0-586c650d77c8-7201/screen/7177f4d5-995e-4906-ad80-1d7fe4e9e08e/Apply-Dependents-Add-Child-2)

Reply:
Fixed